### PR TITLE
manual: make compile and fixup tosi chapter

### PR DESCRIPTION
**Pull request by tjhei**

### CODE REVIEW

File: doc/manual/manual.tex [hunk 1]
```diff
@@ -9967,11 +9967,11 @@ \subsubsection{The Tosi et al.~benchmarks}
 All five benchmarks solve for Boussinesq convection in a box of $1 \times 1$ dimensions with free slip boundary conditions. The initial temperature distribution considers a linear depth profile with a slight perturbation to start convection. Top and bottom boundaries are set to a fixed temperature value. The parameters shared between the benchmark cases can be found in \url{benchmarks/tosi_et_al_2015_gcubed/Tosi_base.prm}. The other input files describe the variations on this base model, which pertain to the rheological description. The specific rheologies used are implemented in \url{benchmarks/tosi_et_al_2015_gcubed/tosi.cc} and describe a linear and a plastic component of the viscosity:
 \begin{align}
   \eta_\text{linear}(T,z) &= \exp(-\ln(\eta_T) T + \ln(\eta_Z) z)
-  \label{eq:tosi-benchmark-lin-visc}
-  \eta_\text{plastic}(\dot\epsilon) &= \eta^\asterisk + \frac{\sigma_y}{\sqrt(\dot\epsilon:\dot\epsilon)}
+  \label{eq:tosi-benchmark-lin-visc} \\
+  \eta_\text{plastic}(\dot\epsilon) &= \eta^* + \frac{\sigma_y}{\sqrt(\dot\epsilon:\dot\epsilon)}
```
Comment (bangerth) [inline doc/manual/manual.tex]: ?? The two are printed differently. What do you not like about this?

Comment (tjhei) [reply doc/manual/manual.tex]: before: 
error: \asterisk is undeclared
error: you can not have two labels in a single line

which of the two errors do you want to fix in a different way?

Comment (tjhei) [reply doc/manual/manual.tex]: do you want me to use \ast instead of *?

Comment (bangerth) [reply doc/manual/manual.tex]: Ah, `\ast`, yes. Please use that instead. The other change is obviously correct.

**Comment by tjhei** [comment]
fixed! (from SFO)

**Comment by bangerth** [comment]
Merged (from DEN) :-)

**Pull request merged**